java/src/jmri/jmrix/srcp/SRCPMessage.java

```diff
@@ -33,137 +33,83 @@ public SRCPMessage(String m) {
-        m.setElement(5, ' ');
-        m.addIntAsFourHex(LocoAddress.getNumber(), 6);
-        return m;
-    }
-
-    /* 
-     * get a static message to add a locomotive to a standard consist in 
-     * the reverse direction
-     * @param ConsistAddress - a consist address in the range 1-255
-     * @param LocoAddress - a jmri.DccLocoAddress object representing the 
-     * locomotive to add
-     * @return an SRCPMessage of the form GS cc llll 
-     */
-    static public SRCPMessage getAddConsistReverse(int ConsistAddress, jmri.DccLocoAddress LocoAddress) {
-        SRCPMessage m = new SRCPMessage(10);
-        m.setBinary(false);
-        m.setOpCode('G');
-        m.setElement(1, 'R');
-        m.setElement(2, ' ');
-        m.addIntAsTwoHex(ConsistAddress, 3);
-        m.setElement(5, ' ');
-        m.addIntAsFourHex(LocoAddress.getNumber(), 6);
-        return m;
-    }
-
-    /* 
-     * get a static message to subtract a locomotive from a Standard Consist
-     * @param ConsistAddress - a consist address in the range 1-255
-     * @param LocoAddress - a jmri.DccLocoAddress object representing the 
-     * locomotive to remove
-     * @return an SRCPMessage of the form GS cc llll 
-     */
-    static public SRCPMessage getSubtractConsist(int ConsistAddress, jmri.DccLocoAddress LocoAddress) {
-        SRCPMessage m = new SRCPMessage(10);
-        m.setBinary(false);
-        m.setOpCode('G');
-        m.setElement(1, 'S');
-        m.setElement(2, ' ');
-        m.addIntAsTwoHex(ConsistAddress, 3);
-        m.setElement(5, ' ');
-        m.addIntAsFourHex(LocoAddress.getNumber(), 6);
-        return m;
-    }
-
-    /* 
-     * get a static message to delete a standard consist
-     * @param ConsistAddress - a consist address in the range 1-255
-     * @return an SRCPMessage of the form GK cc 
-     */
-    static public SRCPMessage getKillConsist(int ConsistAddress) {
-        SRCPMessage m = new SRCPMessage(5);
-        m.setBinary(false);
-        m.setOpCode('G');
-        m.setElement(1, 'K');
-        m.setElement(2, ' ');
-        m.addIntAsTwoHex(ConsistAddress, 3);
-        return m;
-    }
 
-    /* 
-     * get a static message to display a standard consist
-     * @param ConsistAddress - a consist address in the range 1-255
-     * @return an SRCPMessage of the form GD cc 
+    /*
+     * @param bus - a bus number
+     * @return an SRCPMessage to initialize programming on the given bus.
      */
-    static public SRCPMessage getDisplayConsist(int ConsistAddress) {
-        SRCPMessage m = new SRCPMessage(5);
-        m.setBinary(false);
-        m.setOpCode('G');
-        m.setElement(1, 'D');
-        m.setElement(2, ' ');
-        m.addIntAsTwoHex(ConsistAddress, 3);
-        return m;
-    }
-
     static public SRCPMessage getProgMode(int bus) {
         String msg = "INIT " + bus + " SM NMRA\n";
         SRCPMessage m = new SRCPMessage(msg);
         return m;
     }
 
+    /*
```

Comment:
Are these supposed to be JavaDocs or just inline comments? If JavaDocs, the comment needs to start with two stars.